### DISCUSSION TITLE
Ops.Gl.Meshes.Circle jumpy percent animation

### DISCUSSION THREAD

**Discussion by tgd-git** [cables-gl/cables_docs#254] [Answers & Questions]
hey, i can not figure out how to smoothly animate the percent value of the circle. i am using boolanim. the values seem smooth but the visual is quite rough and jumpy. is there any way to improve that?

**Comment by steam0r** [comment]
hey, do you have a simple example? this seems to be fine for me. did you check your easing of boolanim?

![Bildschirmfoto 2022-05-31 um 11 48 02](https://user-images.githubusercontent.com/151271/171145794-002571ae-38be-4f01-82bd-81f98232fce6.png)

**Reply by tgd-git** [reply]
your screenshot is pretty much the example. just set the duration higher like 3 sec. also compare to the animation of radius. 
![image](https://user-images.githubusercontent.com/2562637/171155389-452c03b2-f62f-4e5a-89e0-2a1dbd847691.png)

**Reply by steam0r** [reply]
ah, you can play with the amount of segments of the circle then, increasing that should give you a smoother animation

**Reply by tgd-git** [reply]
yeah but it feels not right and needs crazy large segment counts on higher timeframes, and even then you can still see jumps. 
but i guess thats how the percent works. just had a look at the animation in wireframe and it animates by removing segments and not by "contracting" the vertices.

**Comment by pandrr** [comment]
here is a pixel animation based one: https://cables.gl/edit/jfrTp4

**Reply by tgd-git** [reply]
def cool workaround, thanks but unfortunately i need the mesh itself modified because i want to extrude it.

**Comment by pandrr** [comment]
then you have to write your own
